EIPS/eip-770.md

```diff
@@ -0,0 +1,124 @@
+## Preamble
+
+    EIP: 770 <or to be assigned by an editor>
+    Title: Decentralized Booking Protocol Standard
+    Author: Vidal Chriqui <vidal@btuprotocol.com>
+    Type: Standard
+    Category : ERC
+    Status: Draft
+    Created: 2017-12-26
+    Requires : ERC 20
```

Comment:
An extra space before `:`.

`Requires: 20`
should be fine.

Reply:
Will be done in next submission